scripts/facts.runs/FACTS_TROUBLESHOOTING.md

```diff
@@ -52,3 +58,69 @@ conda activate dscim-facts-epa
 Now you are ready to run FACTS again.
 
 </details>
+
+<details>
+
+<summary><b>FACTS crashes shortly after "Setting up ZMQ queues"</b></summary>
+
+Problem: 
+
+When running FACTS in a container, the run fails shortly after the output message "Setting up ZMQ queues". Initial exceptions in the stack trace vary but usually the stack trace ends with something like:
+```
+The above exception was the direct cause of the following exception:
+
+Traceback (most recent call last):
+  File "runFACTS.py", line 193, in <module>
+    run_experiment(args.edir, args.debug, args.alt_id, resourcedir=args.resourcedir, makeshellscript = args.shellscript, globalopts = args.global_options)
+  File "runFACTS.py", line 86, in run_experiment
+    amgr.run()
+  File "/usr/local/lib/python3.8/dist-packages/radical/entk/appman/appmanager.py", line 485, in run
+    raise EnTKError(ex) from ex
+radical.entk.exceptions.EnTKError
+```
+
+Potential Solution: 
```

Comment:
@JMGilbert can you clarify in this section where the commands should be run -- should they be in the docker terminal or outside the docker?